charts/kube-ovn/templates/ovsovn-ds.yaml

```diff
@@ -81,18 +81,7 @@ spec:
           {{- if .Values.DPDK }}
           command: ["/kube-ovn/start-ovs-dpdk.sh"]
           {{- else }}
-          command:
-          {{- if .Values.DISABLE_MODULES_MANAGEMENT }}
```

Comment:
Why we introduced `DISABLE_MODULES_MANAGEMENT`? Is it related to https://github.com/kubeovn/kube-ovn/issues/4286？

Reply:
Because it is the only way for running kube-ovn on Talos Linux
https://github.com/kubeovn/kube-ovn/blob/ed16ce5a2b8fb4288c88d87fefb174530796a007/charts/kube-ovn/README.md?plain=1#L24

Reply:
I have deployed my simple ovs daemonset inside talos and works fine. The problem is : ```modinfo``` is lying because it fetches the list of known modules by reading the ```/lib/modules/$(uname -r)/modules.*``` files, which are usually updated with ```depmod```.

```depmod -a``` can't run in talos and modinfo does not knows about it. This does not prevent us from loading the module.